src/net.rs

```diff
@@ -260,11 +263,9 @@ impl RedisManager {
                     })?;
                     Ok(())
                 }
-                match inner_subscribe(id, client, message_processor, &channel_name) {
-                    Ok(()) => {}
-                    Err(e) => error_sender
-                        .send(e)
-                        .expect("Can't notify RedisManager connection error"),
+                let _ = match inner_subscribe(id, client, message_processor, &channel_name) {
+                    Ok(()) => Ok(()),
+                    Err(e) => error_sender.send(e),
                 };
```

Comment:
```suggestion
                if let Err(e) = inner_subscribe(id, client, message_processor, &channel_name) {
                    let _ = error_sender.send(e);
                }
```

Reply:
I think using `if let` is more simple.

Reply:
45d3efb4094b38bf2e2a76b1f7120db975497fe1